src/main/java/ee/tuleva/onboarding/aml/AmlAutoChecker.java

```diff
@@ -67,14 +70,30 @@ public void beforeMandateCreated(BeforeMandateCreatedEvent event) {
     Address address = event.getAddress();
 
     if (event.isThirdPillar()) {
-      amlService.addSanctionAndPepCheckIfMissing(user, address);
+      if (isThirdPillarMandateAmlCheckRequired(user, event.getMandate())) {
+        amlService.addSanctionAndPepCheckIfMissing(user, address);
+      }
     }
 
     if (!amlService.allChecksPassed(user, pillar)) {
       throw AmlChecksMissingException.newInstance();
     }
   }
 
+  private Boolean isThirdPillarMandateAmlCheckRequired(User user, Mandate mandate) {
+    var conversion = userConversionService.getConversion(user).getThirdPillar();
+    var isTulevaThirdPillarClient =
+        conversion.isPartiallyConverted() || conversion.isFullyConverted();
```

Comment:
`conversion.isPartiallyConverted()` would suffice, but not harm doing both checks either.